script/DeployVault.sol

```diff
@@ -0,0 +1,250 @@
+import { DelayedWithdrawalManager } from "src/delayed-withdrawal-manager/DelayedWithdrawalManager.sol";
+import { TOSManager } from "src/strategies/layers/creation-validation/external/TOSManager.sol";
+import { GlobalValidationManagersRegistry } from
+  "src/strategies/layers/creation-validation/external/GlobalValidationManagersRegistry.sol";
+import { SignatureBasedWhitelistManager } from
+  "src/strategies/layers/creation-validation/external/SignatureBasedWhitelistManager.sol";
+import { ExternalFirewall } from "@forta/firewall/ExternalFirewall.sol";
+import {
+  FirewallAccess,
+  FIREWALL_ADMIN_ROLE,
+  PROTOCOL_ADMIN_ROLE,
+  ATTESTER_MANAGER_ROLE,
+  CHECKPOINT_EXECUTOR_ROLE,
+  TRUSTED_ATTESTER_ROLE
+} from "@forta/firewall/FirewallAccess.sol";
+import { ISecurityValidator } from "@forta/firewall/SecurityValidator.sol";
+import { FirewallRouter } from "@forta/firewall/FirewallRouter.sol";
+
+import { ICheckpointHook } from "@forta/firewall/interfaces/ICheckpointHook.sol";
+import { Checkpoint, Activation } from "@forta/firewall/interfaces/Checkpoint.sol";
+
+import { BaseDeploy } from "./BaseDeploy.sol";
+import { console2 } from "forge-std/console2.sol";
+
+contract DeployVault is BaseDeploy {
+  function run() external {
+    address signer = vm.envAddress("SIGNER");
+    vm.startBroadcast();
+    console2.log("Deployer:", deployer);
+    address strategyRegistry =
+      deployContract("V1_STRATEGY_REGISTRY", abi.encodePacked(type(EarnStrategyRegistry).creationCode));
+    console2.log("Strategy registry:", strategyRegistry);
+    address nftDescriptor = deployContract(
+      "V1_NFT_DESCRIPTOR",
+      abi.encodePacked(
+        type(EarnNFTDescriptor).creationCode, abi.encode("https://api.balmy.xyz/v1/earn/metadata/", admin)
+      )
+    );
+    console2.log("NFT descriptor:", nftDescriptor);
+    address[] memory initialAdmins = new address[](2);
+    initialAdmins[0] = admin;
+
+    // FORTA
+    bytes32 attesterControllerId = bytes32("3");
+    ISecurityValidator validator = ISecurityValidator(0xc9b1AeD0895Dd647A82e35Cafff421B6CcFe690C);
+
+    address firewallAccess =
+      deployContract("V1_FACCESS", abi.encodePacked(type(FirewallAccess).creationCode, abi.encode(deployer)));
+    address externalFirewall = deployContract(
+      "V1_FEXTERNAL",
+      abi.encodePacked(
+        type(ExternalFirewall).creationCode,
+        abi.encode(validator, ICheckpointHook(address(0)), attesterControllerId, firewallAccess)
+      )
+    );
+    address firewallRouter = deployContract(
+      "V1_FROUTER",
+      abi.encodePacked(
+        type(FirewallRouter).creationCode,
+        abi.encode(ExternalFirewall(externalFirewall), FirewallAccess(firewallAccess))
+      )
+    );
+    address vault = deployContract(
+      "V1_VAULT",
+      abi.encodePacked(
+        type(FirewalledEarnVault).creationCode,
+        abi.encode(IEarnStrategyRegistry(strategyRegistry), admin, initialAdmins, nftDescriptor, firewallRouter)
```

Comment:
I just realized that even though the `admin` should be the msig, the `initialPauseAdmins` should be the governor. So that we can react quicker